ArduPlane/ArduPlane.cpp

```diff
@@ -963,6 +963,28 @@ bool Plane::flight_option_enabled(FlightOptions flight_option) const
     return g2.flight_options & flight_option;
 }
 
+#if AP_SCRIPTING_ENABLED
+// this implements the virtual function defined in AP_Vehicle for Plane to set the desired airspeed in m/s
+bool Plane::set_desired_airspeed(float airspeed_new)
+{
+    plane.new_airspeed_cm = constrain_float(airspeed_new, aparm.airspeed_min, aparm.airspeed_max) * 100.0f;
```

Comment:
I wonder if we should return false if outside the valid range and not do anything. I'm not sure...

Reply:
Looks like this only works in guided and auto, so we should return false in any other mode.

Reply:
Done